src/server/services/wcs/qgswcsgetcapabilities.h

```diff
@@ -27,13 +27,15 @@ namespace QgsWcs
   /**
    * Create get capabilities document
    */
-  QDomDocument createGetCapabilitiesDocument( QgsServerInterface* serverIface, const QString& version,
+  QDomDocument createGetCapabilitiesDocument( QgsServerInterface* serverIface,
+      const QgsProject* project, const QString& version,
       const QgsServerRequest& request );
 
   /** Output WCS  GetCapabilities response
    */
-  void writeGetCapabilities( QgsServerInterface* serverIface, const QString& version,
-                             const QgsServerRequest& request, QgsServerResponse& response );
+  void writeGetCapabilities( QgsServerInterface* serverIface, const QgsProject* project,
+                             const QString& version, const QgsServerRequest& request,
+                             QgsServerResponse& response );
```

Comment:
* .vs. & -> to me ptr (smart to indicate who is resp. for the resource) indicate the possibility for null, I'd use & everywhere else. Is the function able to do something meaningfull without the serverIface... maybe, without a project... maybe. If its the case, doc, if it's not &